src/test/java/com/crowdin/client/translations/TranslationsApiTest.java

```diff
@@ -84,7 +76,7 @@ public void listProjectBuildsTest() {
 
     @Test
     public void buildProjectTranslationTest() {
-        BuildProjectTranslationRequest request = new BuildProjectTranslationRequest();
+        CrowdinTranslationCreateProjectBuildForm request = new CrowdinTranslationCreateProjectBuildForm();
```

Comment:
Please add additional test with `CrowdinTranslationCraeteProjectPseudoBuildForm` request body and with definied `charTransformation` field

Reply:
Added